meganotes-master/app/notes/notes.service.js

```diff
@@ -0,0 +1,71 @@
+(function() {
```

Comment:
TODO: More bindable members at top (in this case instead of vm you'd use service) for getNotes, create, update, etc.